### PR TITLE
Remove overrides to avoid self-DOSsing

### PR DESCRIPTION
**Pull request by coverbeck**
**Description**
Now that self-DOSsing bug has been fixed, we no longer need the test overrides that prevented the triggering of it.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-4767

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.

### CODE REVIEW

**Comment by svonworl** [comment]
Should we target 1.13.x?

**Comment by coverbeck** [comment]
> Should we target 1.13.x?

Not branch-wise, if that's what you mean. Smoke tests, a bit confusingly, are always run off the develop branch.

File: cypress/integration/smokeTests/prodOnly/search.ts [hunk 1]
```diff
@@ -1,64 +0,0 @@
-/**
- * Remove this file after 1.13 deployed and update package.json "test-staging-no-auth"
- * and "test-prod-no-auth" to restore the sharedTests version of this file.
```
Comment (kathy-t) [inline cypress/integration/smokeTests/prodOnly/search.ts:3]: Do you need to update the package.json?

**Review by svonworl — APPROVED** [review]

File: cypress/integration/smokeTests/prodOnly/search.ts (hunk 1 above)
Comment (coverbeck) [reply cypress/integration/smokeTests/prodOnly/search.ts:3]: Good catch, all I needed to do was read my own comment.

**Pull request merged**